src/components/LogWindow/index.tsx

```diff
@@ -16,27 +15,11 @@ const LogWindow: React.FC<LogWindowProps> = ({ logType, isUploadedLog }) => {
   const {
     clearExpandedLines,
     collapseLines,
-    expandLines,
-    getLine,
-    getResmokeLineColor,
-    resetRowHeightAtIndex,
     scrollToLine,
-
     expandedLines,
-    highlightedLine,
     lineCount,
-    preferences,
     processedLogLines,
-    range,
-    searchState,
   } = useLogContext();
-  const { prettyPrint, wrap } = preferences;
-  const { searchTerm } = searchState;
-
-  const [highlights] = useHighlightParam();
-  const highlightRegex = highlights.length
-    ? new RegExp(highlights.join("|"), "i")
-    : undefined;
```

Comment:
I’m questioning if this is true, since `RowRenderer` is actually a singular component with a for-loop inside of it. Since the highlight regex is declared outside of the for-loop, I think it only gets defined once. I checked this manually to the best of my ability, but let me know if I missed something.

Reply:
Great call I hadn't noticed that!